compute_sdk/tests/unit/test_client.py

```diff
@@ -56,16 +57,6 @@ def test_client_init_sets_addresses_by_env(
     assert client.funcx_service_address == web_uri
 
 
-def test_client_init_accepts_specified_taskgroup():
-    tg_uuid = uuid.uuid4()
-    gcc = gc.Client(
-        task_group_id=tg_uuid,
-        do_version_check=False,
-        login_manager=mock.Mock(),
-    )
-    assert gcc.session_task_group_id == str(tg_uuid)
-
```

Comment:
I don't see that we removed anything related to this argument or functionality; is there another reason to remove this test?

Reply:
Heh.  Looks like we crossed paths: I added the review comment, asked you about it, you updated it, and then I submitted.  This is addressed now.